go.mod

```diff
@@ -1,146 +1,149 @@
 module github.com/rancher/webhook
 
-go 1.19
+go 1.20
 
 // on release remove this wrangler replace and use the latest tag
 replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230705223603-201b4da5bdaf
```

Comment:
Can you also update wrangler to this `github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df`